src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/masonry-decorator/masonry-view.js

```diff
@@ -447,7 +448,13 @@ define([
              */
             itemLoadedHandler: function($item) {
                 this.sandbox.masonry.refresh('#' + this.masonryGridId, true);
-                this.sandbox.dom.removeClass($item, constants.loadingClass);
+                $item.removeClass(constants.loadingClass);
+                // lock the height of the image container
+                $item.find('.masonry-image').height($item.find('.masonry-image').height());
```

Comment:
The `itemLoadedHandler` gets executed when everything inside a masonry item, including its image, has been loaded. The height of the image-container needs to be fixed, to make the zoom effect on hover possible.